codegen/smithy-aws-go-codegen/src/main/resources/software/amazon/smithy/aws/go/codegen/customization/apigw_exports_exceptions/com.amazonaws.route53recoverycontrolconfig#Route53RecoveryControlConfig.exceptions.json

```diff
@@ -0,0 +1,43 @@
+{
+    "notpicked": [
+        "smithy.api#cors$maxAge",
+        "smithy.api#cors$origin",
+        "smithy.api#http$code",
+        "smithy.api#idRef$selector",
+        "smithy.api#PrimitiveBoolean",
+        "smithy.api#PrimitiveByte",
+        "smithy.api#PrimitiveDouble",
+        "smithy.api#PrimitiveFloat",
+        "smithy.api#PrimitiveInteger",
+        "smithy.api#PrimitiveLong",
+        "smithy.api#PrimitiveShort",
+        "smithy.api#TraitDiffRule$severity",
+        "smithy.waiters#Waiter$maxDelay",
+        "smithy.waiters#Waiter$minDelay"
+    ],
+    "picked": [
+        "com.amazonaws.route53recoverycontrolconfig#__boolean",
```

Comment:
The idea here is that this is a snapshot of the current set of members that have an incorrect or most likely incorrect default value, and when new members are added, they won't have an incorrect default? If so, that sounds good. You'll still need to ensure that all members that target root-level shapes that had their defaults added back in this file also have their defaults added back, otherwise the model will be invalid (maybe you're already doing that).